packages/mobile/src/home/WalletHome.tsx

```diff
@@ -105,6 +105,7 @@ export class WalletHome extends React.Component<Props, State> {
 
     ValoraAnalytics.setUserAddress(this.props.address)
 
+    this.props.refreshAllBalances()
```

Comment:
Nit: This could be more impactful if it triggers on an event listener like `AppState` or `onFocus` but this is a good improvement nonetheless

Reply:
This is a great idea. I'm not sure how to do that with a class component, but I added a TODO and left a code snippet for whenever we move this to a function component.

Reply:
Definitely easier to do using hooks but here's how I did it for app background/foreground if you're interested: https://github.com/celo-org/celo-monorepo/blob/da677b6b024a4c6e3c60917ceb9b44c56eac15e3/packages/sdk/identity/src/odis/phone-number-identifier.ts#L101

Reply:
I will try to prioritize moving some screens to functions soon. It's relatively easy to do and will help have our codebase in a better state, it's always annoying when I run into a screen that uses the old way and it makes reviewing PRs that include those changes more complicated.

Reply:
Yeah my bad: https://github.com/celo-org/wallet/pull/286/files#diff-aee5619c99fc9bb7d938ef5bc2b6e93234a97b412ad4a7857e5e504a4f581340R31-R39

Agree that it's not a huge deal and the better solution is to work on refactoring old class components